Dockerfile

```diff
@@ -50,4 +56,10 @@ RUN make installcheck
 ###
 # this creates a usable postgres image but without the packages needed to build
 FROM base AS output
+
+# Automatically enable pg_duckdb
+RUN echo "shared_preload_libraries='pg_duckdb'" >> /usr/share/postgresql/postgresql.conf.sample
+RUN echo "CREATE EXTENSION IF NOT EXISTS pg_duckdb;" >> /docker-entrypoint-initdb.d/0001-install-pg_duckdb.sql
```

Comment:
This is probably okay to do here. I was only hesitant to do so in case some users want to opt out of this behavior, doing so this way means they need to make their own image which removes this file. But if we get push back it can always be removed.